static/style.css

```diff
@@ -124,13 +127,41 @@ header h2 {
     border-top: 2px solid #262119ff;
     border-bottom: 2px solid #262119ff;
 }
+#hamburger {
+    display: none;
+}
+
 @media only screen and (max-width: 600px) {
-    nav ol {
+    #hamburger {
+        display:inline-block;
+        position: fixed;
+        top: 0px;
+        right: 2%;
+        z-index: 2;
+        padding: 4%;
+        border: solid 1px #53462bff;
+        border-radius: 30%;
+        font-size: 1.5rem;
+        font-weight: 800;
+        color: #53462bff;
+    }
+    #hamburger:hover {
+        cursor: pointer;
+    }
+    .navbar {
+        display: none;
+        position: fixed;
+        top: 0px;
+        right: 10%;
+        z-index: 1;

```

Comment:
z-index is useful here, but be careful about overusing it. Lean on just structuring your dom elements in the right order when you can. You can get into places where different people are setting different z-indexes everywhere and its impossible to get things to be in the right order. 

I've been at 3 different companies that fucked this up in a super annoying way. The "right" solution though is complicated, where you implement a proper "layer" system using [stacking contexts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) correctly (these are unfortunately annoyingly complicated and near-impossible to remember how they work until you have a problem that demands you go figure it out). As such don't worry about that for now, but just be aware that this can be a problem if overused in a team setting.